waves-ext/src/main/scala/com/wavesplatform/dex/grpc/integration/services/WavesBlockchainApiGrpcService.scala

```diff
@@ -66,6 +68,13 @@ class WavesBlockchainApiGrpcService(context: ExtensionContext)(implicit sc: Sche
     utxChangesSubscribers.clear()
   }
 
+  private val inUtxHandler: exchange.ExchangeTransaction => Future[CheckedBroadcastResponse.Result] = {
+    if (context.settings.config.as[Boolean]("waves.extension.dex.allow-tx-rebroadcasting"))
```

Comment:
1. Please use https://github.com/wavesplatform/Waves/blob/master/node/src/main/resources/application.conf#L263
2. Also we need to cache this value in the class and use the cached value

Reply:
Fixed